spec.bs

```diff
@@ -632,9 +601,137 @@ To <dfn>obtain a report's shared info</dfn> given an [=aggregatable report=]
 1. Return the result of [=serializing an infra value to a json string=] given
     |sharedInfo|.
 
+Shared Storage API monkeypatches {#shared-storage-api-monkeypatches}
+====================================================================
+
+Issue(43): This should be moved to the Shared Storage spec.
+
+<xmp class="idl">
+partial interface SharedStorageWorkletGlobalScope {
+  readonly attribute PrivateAggregation privateAggregation;
+};
+</xmp>
+
+At the start of each invocation of a Shared Storage operation (i.e. a call to
+`run()` or to `selectURL()`), create a new [=batching scope=] and associate it
+with that invocation.
+
+When a {{PrivateAggregation}} object is exposed on a
+{{SharedStorageWorkletGlobalScope}}, its [=PrivateAggregation/get batching scope
+steps=] should be left default until the `addModule()` execution is complete. At
+that point, it should be set to the following algorithm:
+1. Return the [=batching scope=] associated with the currently executing
+    operation invocation in |scope|.
+
+Note: Multiple operation invocations can be in-progress at the same time, each
+with a different batching scope. However, only one can be currently executing.
+
+When an operation invocation completes, [=mark a batching scope complete=] given
```

Comment:
Same here - is there an algorithm that runs the operation invocation, that can call this logic?